### ISSUE TITLE
Mention Decompiler Explorer GitHub org in FAQ/About

### ISSUE THREAD

**Issue by mborgerson** [decompiler-explorer/decompiler-explorer#29]
Mention:
* This is a collaborative effort, so we should change the 'Made by Vector35' verbiage to reflect it
* Mention the GitHub org
* Update URLs

**Comment by psifertex** [comment]
We should replace that with a "Contributions" section that links to the groups that have contributed:

Code: Angr, V35, etc
Hosting costs: Hex-Rays, V35

I don't think saying it's "the decompiler-explorer organization" as the source actually adds value but rather obfuscates the groups who've actually contributed.

**Comment by psifertex** [comment]
Or maybe each individual who has contributed can choose to have any affiliation listed or not up to their preference. Though for funding for hosting obviously it should be the organizations.

**Comment by mborgerson** [comment]
> I don't think saying it's "the decompiler-explorer organization" as the source actually adds value

Maybe my wording was not clear. We should mention where the project is being developed and who is collaborating to build it (members of the org). And yes, we should mention the companies that are funding the project's hosting and providing licenses for their products.

**Comment by psifertex** [comment]
Yeah, I just wanted to be clear, I think we are on the same page.
